CoreGraphNode/src/au/gov/asd/tac/constellation/graph/node/plugins/DefaultPluginEnvironment.java

```diff
@@ -58,6 +58,10 @@
     @Override
     public Future<?> executePluginLater(final Graph graph, final Plugin plugin, final PluginParameters parameters, final boolean interactive, final List<Future<?>> async, final PluginSynchronizer synchronizer) {
 
+        if (graph == null) {
+            LOGGER.log(Level.WARNING, plugin.getName() + " plugin was executed on a graph which was null");
```

Comment:
getting super picky here but I think NetBeans provides an optimisation here to convert the string concatenation with a message template

`LOGGER.log(Level.WARNING, "{0} plugin was executed on a graph which was null", plugin.getName());`